<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MGetTextDlg</class>
 <widget class="QDialog" name="MGetTextDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MGetTextDlg</string>
  </property>
  <widget class="QLabel" name="TipLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>请输入文本</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>391</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;宋体&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AcceptButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
